frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py

```diff
@@ -189,7 +193,7 @@ def _compute_in_simulation(self, list_patterns: list):
             a_as_int = self.alphabet.encode(a)
             b_as_int = self.alphabet.encode(b)
 
-            l1_simulate = levenshtein_simulate(self.module, a_as_int, b_as_int)
+            l1_simulate = levenshtein_fhe(self.module, a_as_int, b_as_int)
```

Comment:
Why is this changed from simulation to actual fhe?

Reply:
I mean execution is tested right below, simulation is just fast testing. It also shows the users how to use simulation. I think it should stay.

Reply:
no simulation code == real execution code, only the call to execution should differ (simulate vs encrypt_run_decrypt) there is no reason to have 2 codes here.

Reply:
removing from PR